app/routes/groups.tsx

```diff
@@ -1,26 +1,48 @@
-import { Card } from '~/components/ui/containers';
 import { useGroups } from '~/hooks/useGroups';
 import { Link } from '@remix-run/react';
-import { LinkButton } from '~/components/ui/forms';
+import { Button } from '~/components/ui/button';
+import { Image } from '~/components/ui/images';
 
 export default function Group() {
   const groups = useGroups();
   return (
-    <>
-      <Card>
-        <ul className="max-w-md space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
+    <div className="bg-primary">
+      <div className="m-auto max-w-screen-xl">
+        <ul className="w-full flex flex-wrap justify-center text-secondary">
           {groups?.map((group) => {
             return (
-              <li key={group.id}>
-                <Link to={group.id}>{group.name}</Link>
+              <li className="flex flex-wrap justify-center my-2 mx-auto list-none md:m-2" key={group.id}>
+                <Link
+                  style={{ border: '#f5f5f5 1px solid', borderRadius: '1rem' }}
```

Comment:
I think this may be possible to do in Tailwind. Let's see if we can to keep our styling consistent.